walkgen_params:
  planeseg: False  # Use URDF of the environment or planeseg visualisation.
  extract_methodId: 0 # 0 load surfaces from single .stl file and process it or 1 from folder of .stl files (1 .stl = 1 surface).

  world:
    path: ""  # if path = "", path = os.path.dirname(os.path.abspath(__file__))
    urdf: "/data/urdf/lab_scene.urdf"  # Env URDF path + urdf
    heightmap: "/data/lab_scene.dat"  # Heightmap path + heightmap

  params:
    margin_inner: 0.02  # inner surface margin
    margin_outer: 0.04  # outer surface margin
    n_points: 6  # Maximum number of vertices for the surfaces
    method_id: 3  # Method to remove overlapping
    poly_size: 10  # Number of maximum point for convex decomposition.
    min_area: 0.03  # Minimum area to keep the surfaces
    N_phase: 6  # Number of step to proceed (--> N_phase * n_gait step in SL1M)
    N_phase_return: 2  # Number of step to return (N_phase_return surfaces for each foot)
